bitbots_ceiling_cam/config/camera_settings_ceiling_cam.yaml

```diff
@@ -0,0 +1,119 @@
+#  The tf frame under which the images were published
+camera_frame: ceiling_cam
+
+#  The DeviceUserID of the camera. If empty, the first camera found in the
+#  device list will be used
+device_user_id: ""
```

Comment:
Maybe we should add the camera name here, like in the config from @MonsterUntermBett . Otherwise, the driver maybe connects to the wrong camera.

Reply:
i agree